raml-headers-form.html

```diff
@@ -197,7 +223,7 @@
                     <label>Header name</label>
                     <input
                       is="anypoint-input"
-                      class="name-change-input"
+                      class="name-change-input anypoint-text-input"
```

Comment:
The one you show is the same. I was talking about these:

<img width="497" alt="screenshot 2017-05-26 15 33 30" src="https://cloud.githubusercontent.com/assets/7399928/26496647/b5b83fa4-4228-11e7-9bb7-107fc3915c02.png">